pkg/lib/config/sms_gateway.go

```diff
@@ -0,0 +1,36 @@
+package config
```

Comment:
The convention is to name this file `sms_gateway_env.go`. For example, `redis_env.go`.